packages/protocol/package.json

```diff
@@ -6,61 +6,61 @@
   "author": "Celo",
   "license": "LGPL-3.0",
   "scripts": {
-    "lint:ts": "yarn run --top-level eslint .",
-    "lint:sol": "solhint --version && solhint './contracts/**/*.sol' && solhint './contracts-0.8/**/*.sol'",
-    "lint": "yarn run lint:ts && yarn run lint:sol",
+    "anvil-devchain:e2e-tests": "./scripts/foundry/run_e2e_tests_in_anvil.sh",
+    "anvil-devchain:integration-tests": "./scripts/foundry/run_integration_tests_in_anvil.sh",
+    "anvil-devchain:start-L1": "./scripts/foundry/create_and_migrate_anvil_devchain.sh",
+    "anvil-devchain:start-L2": "./scripts/foundry/create_and_migrate_anvil_l2_devchain.sh",
+    "anvil-devchain:status": "if nc -z localhost 8546; then echo 'Devchain is serving at http://localhost:8546'; else echo 'Devchain is not running.'; fi",
+    "anvil-devchain:stop": "./scripts/foundry/stop_anvil.sh",
+    "artifacts:download": "./scripts/bash/download_artifacts.sh",
```

Comment:
Not really sure whether these are needed - it might be worth checking if they are effectively used anywhere

Reply:
We used to keep Truffle build artifacts in GCP buckets, but it looks like we actually stopped using this pattern a long time ago, just a few months after mainnet launch. Looking at the [GCP buckets](https://console.cloud.google.com/storage/browser/contract_artifacts;tab=objects?forceOnBucketsSortingFiltering=true&inv=1&invt=Abv__Q&project=celo-testnet&prefix=&forceOnObjectsSortingFiltering=true&pageState=(%22StorageObjectListTable%22:(%22s%22:%5B(%22i%22:%22objectListDisplayFields%2FtimeLastModified%22,%22s%22:%221%22),(%22i%22:%22displayName%22,%22s%22:%220%22)%5D))) the most recent `rc1` bucket is from July 2020 (I believe that was the bucket for the mainnet deployment). 

These should be safe to remove as far as I can tell, and we could clean up the GCP buckets as well.

Reply:
Something to double check since they are referenced in:
- `celotool`
- `helm-charts`
- `README.md`
- `deploy_release_contracts.sh`
- `set_exchange_rate.sh`

Reply:
So:
- `celotool` - is not used
- `helm-charts` - in this repo are not used
- I removed reference in `README.md`
- scripts `deploy_release_contracts.sh` & `set_exchange_rate.sh` were not used

Resolving 😌